src/features/session/login/login-btn.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { onLogin } from "entities/session";
+import { Button } from "shared/ui";
+
+const LoginButton = () => {
+  return (
+    <Button
+      onClick={() => onLogin()}
+      sx={{ fontSize: "18px", px: 4, width: "300px" }}
```

Comment:
on our project we dont use ''300px'' , you can yse only number - 300

Reply:
i see. changed